apps/external-devices/osvr_server_config.externalvrpn.ART_DTrack_Flystick.json

```diff
@@ -0,0 +1,43 @@
+/* Note, that you will need to add a tracker alias for every body/flystick that you configure through VRPN. This descriptor is configured for one Flystick */
+/* Powerwall` coordinate system is what the system expects and if you use a different system, then you will need to apply transforms (see below)*/
+{
+  "externalDevices": {
+    "/DTrackDevice": {
+      "deviceName": "DTrack",
+      "server": "localhost:3884",
+      "descriptor": {
+        "interfaces": {
+          "tracker": {},
+          "button": {},
+          "analog": {}
+        }
+      }
+    }
+  },
+  "aliases": {
+    "/me/head": {
+      "rotate": {
```

Comment:
There's no need to transform since it's using the powerwall coordinate system. But I left it as an example, in case they use a different one and will need an example how to perform transforms